File: docs/user/en/use/api/example/detached-existing-entity-instantiation.java
final SealedEntity brand = new ExistingEntityBuilder(existingEntity)
	.setAttribute("code", "siemens")
	.setAttribute("name", Locale.ENGLISH, "Siemens")
	.setAttribute("logo", "https://www.siemens.com/logo.png")
	.setAttribute("productCount", 1)
	.toInstance();

File: documentation/user/en/use/api/example/detached-existing-entity-instantiation.cs
// can be wrapped into a builder and updated
ISealedEntity brand = new ExistingEntityBuilder(existingEntity)
	.SetAttribute("code", "siemens")
	.SetAttribute("name", new CultureInfo("en"), "Siemens")
	.SetAttribute("logo", "https://www.siemens.com/logo.png")
	.SetAttribute("productCount", 1)
	.ToInstance();